### PR TITLE
Home page and Drug page updates

### PR DESCRIPTION
**Pull request by alival000**
Emergency button added
Drug page updates

### CODE REVIEW

File: src/main/react-webapp/src/components/Home.js [hunk 1]
```diff
@@ -13,7 +13,14 @@ const Home = () => {
     return(
         <div id="content">
             {/* Drop down emergancy menu */}
-
+            
+            <button style="padding: 15px 32px;  text-align: center;  text-decoration: none;
```
Comment (GMarsh0218) [inline src/main/react-webapp/src/components/Home.js]: You should put the styling in home.css but other than that it looks good.

**Review by GMarsh0218 — APPROVED** [review]

File: src/main/react-webapp/src/components/Home.js (hunk 1 above)
Comment (mcollins42) [reply src/main/react-webapp/src/components/Home.js]: +1

**Review by mcollins42 — APPROVED** [review]

File: src/main/react-webapp/src/components/DrugPage.js [hunk 2]
```diff
@@ -24,8 +24,13 @@ const DrugPage = () => {
         <p><b>Classification:</b> {drugInfo["classification"]}</p>
         <DrugPageDesc title="Description" desc={drugInfo["description"]}/>
         <DrugPageList title="Aliases" listItems={drugInfo["aliases"]}/>
+        <DrugPageDesc title="Safety Description" desc={drugInfo["safetyDescription"]}/>
+        <DrugPageList title="Drug Warning Signs" listItems={drugInfo["drugWarningSigns"]}/>
+        <DrugPageDesc title="Description" desc={drugInfo["description"]}/>
```
Comment (mcollins42) [inline src/main/react-webapp/src/components/DrugPage.js:28]: This looks like a duplicate of description above.

**Review by mcollins42 — CHANGES_REQUESTED** [review]

File: src/main/react-webapp/src/components/DrugPage.js (hunk 2 above)
Comment (alival000) [reply src/main/react-webapp/src/components/DrugPage.js:28]: should be updated now

**Review by mcollins42 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**